### ISSUE TITLE
Connection Issue

### ISSUE THREAD

**Issue by tiNsLeY799** [PocketMine/PocketMine-MP#4222]
I'm not quite sure how to reproduce this, or explain this properly, but I'll get on with it.

Each time I attempt to connect to the server, it prompts me with the message "Server is full!" and the display state is the server is not on/connected etc. However, the server is running, as it is shown on MCPE as a remote world
I'll share (a) screenshot(s). This even occurs to my tablet (that is in 0.15) yet it still prompts me with the same message!

![screenshot_2016-06-24-22-52-13](https://cloud.githubusercontent.com/assets/12755193/16355035/002dda52-3a60-11e6-9a5a-c4a5b86f051b.png)
![screenshot_2016-06-24-22-52-16](https://cloud.githubusercontent.com/assets/12755193/16355040/302565f4-3a60-11e6-91fd-33de0f26b875.png)

I am aware that my version is outdated.. I thought maybe it was the PocketMine directory by changing it's version to 0.15, 0.14.0 and 0.14.3.

In conclusion, I am not exactly sure if this is a client issue, or a PocketMine issue. If I cannot get help here, then at _least_ link me to the forums?

**Comment by jojoe77777** [comment]
> If I cannot get help here, then at least link me to the forums?

https://forums.pocketmine.net
